src/app/core/config.py

```diff
@@ -1,16 +1,20 @@
 import logging
 import os
 
+from dotenv import load_dotenv
 from starlette.datastructures import CommaSeparatedStrings
 
+
+load_dotenv()
+
 API_PORT = 8081
 
 ROOT_PATH = os.getenv("ROOT_PATH", "")
-API_DEBUG = False  # os.getenv("LOG_LEVEL", "").strip().lower() == "debug"
+API_DEBUG = os.getenv("API_DEBUG", "False").strip().lower() == "true"
 ALLOWED_HOSTS = CommaSeparatedStrings(os.getenv("ALLOWED_HOSTS", "*"))
 LOG_LEVEL = int(os.getenv("LOG_LEVEL", logging.INFO))
 
-ELASTICSEARCH_URL = os.getenv("ELASTICSEARCH_URL")
```

Comment:
Why are you doing that?

Reply:
that was an oversight and the reason why i started working on that file :)